### DISCUSSION TITLE
how do I use walletnotify?

### DISCUSSION THREAD

**Discussion by gitloverr** [dogecoin/dogecoin#3012] [Q&A] [answered]
hi everyone,

I would like to run walletnotify for dogecoind 

I added this line in doge.conf file : 

`walletnotify=/home/doge/scripts/my.sh %s` 

and my sh file like this: 

`
F=/home/doge/scripts/notify_log
D=`date +"%Y%m%d%H%M%S"`
echo ${D} - ${1} >> ${F}
`

when I send a doge to my wallet this , my notify_log doc. looks like: 

20220621232630 -
20220621232730 -
20220621233540 -

only has a date,  I mean, triggered in a wallet related transaction but I can't get tx id for  my notify_log document...

how can I do? what is missing??

thank u

**Comment by alamshafil** [comment, marked as answer]
Using this `sh` script worked fine for me:

```sh
F="/home/doge/scripts/notify_log" D=$(date +"%Y%m%d%H%M%S") && (echo $D - $1) >> $F
```
Since I launched directly from the command line, I had to use this: `-walletnotify='./test.sh %s'`
This should not be the issue if you use the `dogecoin.conf` file.
